src/listentry.js

```diff
@@ -0,0 +1,10 @@
+import react from 'react';
+const Row = (input)=>{
+   return (
+    <div>
+        {input.value}
+         <input type="checkbox"></input>
```

Comment:
Wrong indentation?

can you please find out the correct indentation?

Reply:
yes i have gone through the guide and find out we use 2 space indentation for each child element.